### DISCUSSION TITLE
can anyone tell me how do fintuning hifigan after training glow model ?on google colab notebook

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#2006] [General Q&A]
can anyone tell me how do fintuning hifigan after training glow model ?on google colab notebook

**Comment by WeberJulian** [comment]
Try first and ask for help when you're stuck.

**Reply by WeberJulian** [reply]
Start from this but with hifigan recipe and your data https://gist.github.com/erogol/97516ad65b44dbddb8cd694953187c5b

**Reply by omkarade** [reply]
I search a lot but I cant find how do fintuning hifigan after training glow model ? please help me @WeberJulian

**Reply by omkarade** [reply]
@WeberJulian I search a lot but I cant find how do fintuning hifigan after training glow model ? please help me @WeberJulian

**Reply by p0p4k** [reply]
There's a recipe in the code base which you can refer to and change for your dataset.
